samza-shell/src/main/bash/run-class.sh

```diff
@@ -153,7 +153,7 @@ fi
 [[ $JAVA_OPTS != *-Xmx* ]] && JAVA_OPTS="$JAVA_OPTS -Xmx768M"
 
 # Check if the GC related flags are specified. If not - add the respective flags to JVM_OPTS.
-[[ $JAVA_OPTS != *PrintGCDateStamps* && $JAVA_OPTS != *-Xloggc* ]] && JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCDateStamps -Xloggc:$SAMZA_LOG_DIR/gc.log"
+[[ $JAVA_OPTS != *-Xloggc* ]] && JAVA_OPTS="$JAVA_OPTS -Xloggc:$SAMZA_LOG_DIR/gc.log"
```

Comment:
PrintGCDateStamps is removed from Java11, so just removing it from here.

Reply:
Can we make it conditional? i.e. if JAVA_HOME is 8, use this, if JAVA_HOME is 11, don't use this?

Reply:
Sure. Just added the conditional for it.